charts/airbyte/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
Nice!

Reply:
https://helm.sh/docs/chart_template_guide/notes_files/ 

So after spinning up people can get instructions to connect.

Reply:
neat!